wiki/People/Centurions/fr.md

```diff
@@ -40,4 +42,4 @@ Les Centurions sont des utilisateurs qui ont atteint la prestigieuse étape de c
 ## Le saviez-vous ?
 
 - Le premier utilisateur à avoir atteint 100 beatmaps classés est ::{ flag=TH }:: [DJPop](https://osu.ppy.sh/users/2363), le 4 août 2010.
-- Parmi tous les Centurions, il faut environ **4 ans et demi**<!-- use "3 ans et demi" instead of "3,5 ans" --> (environ 1668 jours) pour classer 100 beatmaps en moyenne.<!--Stats from https://docs.google.com/spreadsheets/d/1UK1QdaZawxJ75Q0ZwvSGb4zDcC19RumO7BwV0p9R-jM/edit?usp=sharing-->
+- Parmi tous les Centurions, il faut environ **4,8**<!-- use "3 ans et demi" instead of "3,5 ans" --> (environ 1750 jours) pour classer 100 beatmaps en moyenne.<!--Stats from https://docs.google.com/spreadsheets/d/1UK1QdaZawxJ75Q0ZwvSGb4zDcC19RumO7BwV0p9R-jM/edit?usp=sharing-->
```

Comment:
```suggestion
- Parmi tous les Centurions, il faut environ **4,8**<!-- use "3 ans et demi" instead of "3,5 ans" --> (environ 1751 jours) pour classer 100 beatmaps en moyenne.<!--Stats from https://docs.google.com/spreadsheets/d/1UK1QdaZawxJ75Q0ZwvSGb4zDcC19RumO7BwV0p9R-jM/edit?usp=sharing-->
```